### PR TITLE
Rdm 3433 bug fix

### PR DESCRIPTION
**Pull request by mohammedanas**
https://tools.hmcts.net/jira/browse/RDM-3433

Missed casetypetab validation added

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/displaygroup/DisplayGroupDisplayContextParamValidator.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+            validationResult.addError(new ValidationError("Display context parameter is not of type collection") {
+            });
+        }
+        if (StringUtils.isNotBlank(entity.getDisplayContextParameter())) {
+
+            if (isFieldTypeNotTableOrList(entity)) {
+                validationResult.addError(new ValidationError("DisplayContextParameter text should begin with #LIST( or #TABLE("){});
+            } else {
+                String removeBeginingSection = entity.getDisplayContextParameter().indexOf(LIST_PREFIX) > -1
```
Comment (tzarsmango) [inline domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/displaygroup/DisplayGroupDisplayContextParamValidator.java]: syntax error in `beginning`

File: domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/displaygroup/DisplayGroupDisplayContextParamValidator.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import uk.gov.hmcts.ccd.definition.store.repository.entity.DisplayGroupCaseFieldEntity;
+
+@Component
+public class DisplayGroupDisplayContextParamValidator implements DisplayGroupCaseFieldValidator {
+
+    public static final String LIST_PREFIX = "#LIST(";
+    public static final String TABLE_PREFIX = "#TABLE(";
+
+    @Override
+    public ValidationResult validate(DisplayGroupCaseFieldEntity entity) {
```
Comment (andrewfolga) [inline domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/displaygroup/DisplayGroupDisplayContextParamValidator.java]: Just a question but shouldn't we also check if all nested fields in collection are simple type field? Wasn't that agreed with @amansinghb in the end?

Comment (mohammedanas) [reply domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/displaygroup/DisplayGroupDisplayContextParamValidator.java]: Yeah, but that and a few things will come as part of separate ticket, this piece was missing from the original ticket. hence added this.

Comment (andrewfolga) [reply domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/displaygroup/DisplayGroupDisplayContextParamValidator.java]: Thanks for clarification 👍

**Pull request merged**